src/Templates/src/templates/maui-contentview-xaml/.template.config/template.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "http://json.schemastore.org/template",
+    "author": "Microsoft",
+    "classifications": [ "MAUI", "Android", "iOS", "macOS", "Mac Catalyst", "WinUI", "Xaml", "Code" ],
+    "identity": "Microsoft.Maui.ContentView",
```

Comment:
See this comment: https://github.com/dotnet/maui/pull/2065#discussion_r688642467